### PR TITLE
ZEPPELIN-3595. Remove interpreter binding in backend

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?

This PR is trying to remove backend of interpreter binding related code in zeppelin server side. The next step is to put default interpreterSetting into note.json. 

### What type of PR is it?
[Refactoring]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-3595

### How should this be tested?
* CI pass

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by jongyoul** [comment]
Five hundred lines' difficult and sophisticated code will be removed. :-)

File: zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/socket/Message.java [hunk 1]
```diff
@@ -165,13 +165,11 @@
     LIST_NOTE_JOBS,               // [c-s] get note job management information
     LIST_UPDATE_NOTE_JOBS,        // [c-s] get job management information for until unixtime
     UNSUBSCRIBE_UPDATE_NOTE_JOBS, // [c-s] unsubscribe job information for job management
-                                  // @param unixTime
+
     GET_INTERPRETER_BINDINGS,     // [c-s] get interpreter bindings
```
Comment (zjffdu) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/socket/Message.java]: need to remove this message after frontend change

**Comment by jongyoul** [comment]
@felixcheung It's not for now if you set `spark interpreter group` as the first interpreter group in your `zeppelin-site.xml`. I think it needs more implementation even in front side as well to keep the default interpreter setting per note. We need to know whole plan to remove interpreter binding option

**Comment by Tagar** [comment]
@jongyoul I like the idea of having a default interpreter (and not a default interpreter group ) **per note**. 
+1

**Comment by zjffdu** [comment]
Will merge it if no more comments

**Pull request closed without merging**